cogs/hangman.py

```diff
@@ -0,0 +1,293 @@
+import discord
+import string
+from discord.ext import commands
+from discord import app_commands
+from wonderwords import RandomWord
+
+
+class hangmanCog(commands.Cog):
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @app_commands.command(name="play-hangman", description="Play a game of Hangman!")
+    @app_commands.describe(
+        category="Choose a category for the random word.",
+        word_length="The length of the random word as an integer or 'Random' for any word length. (e.g. 7)",
+    )
+    @app_commands.choices(
+        category=[
+            app_commands.Choice(name="Noun", value=1),
+            app_commands.Choice(name="Adjective", value=2),
+            app_commands.Choice(name="Verb", value=3),
+            app_commands.Choice(name="Random", value=4),
+        ],
+    )
+    async def playHangman(
```

Comment:
change to snake case

Reply:
Resolving as the project currently uses camelCase.